src/main/java/org/apache/mesos/hdfs/state/PersistentState.java

```diff
@@ -71,6 +72,22 @@ public void setFrameworkId(FrameworkID frameworkId) throws InterruptedException,
     zkState.store(value).get();
   }
 
+  public void resetDeadNodeTimeStamps() {
+    Date date = DateUtils.addSeconds(new Date(), conf.getDeadNodeTimeout());
+
+    if (getDeadJournalNodes().size() > 0) {
+      deadJournalNodeTimeStamp = new Timestamp(date.getTime());
```

Comment:
This was previously discussed in the following PR, please see the discussion here and outcome, https://github.com/mesosphere/hdfs/pull/94.